### ISSUE TITLE
Why does the double pendulum implementation use 2 Joint Actuators?

### ISSUE THREAD

**Issue by cjds** [guzhaoyuan/drake-tutorial#4]
Looking at the code [here](https://github.com/guzhaoyuan/drake-tutorial/blob/master/drake-controllers/try-out-pid-controller.md#add-source-file), you do 

```
  dp->AddJointActuator("a1", dp->GetJointByName("upper_joint"));
  dp->AddJointActuator("a1", dp->GetJointByName("lower_joint"));
```

However every double pendulum I've seen implemented before has one actuator on the lower joint. If the lower joint and upper joint can be independently controlled, it will make the problem a lot easier to solve IIRC

Questions:

1. Why does the implementation have 2 actuators?
2. How would you edit this code to have only one actuator?

---
TLDR I guess I'm interested in how to leave joints that are completely passive in Drake and wondering how to set that up. Or do I have to set up a Joint Actuator for each joint

**Comment by guzhaoyuan** [comment]
Double pendulum means two-link robot arm. I believe either is okay for this arm to be fully actuated or underactuated (first joint apply no torque). If you want the underactuated version, you could simply apply zero torque on the first joint (set kp ki kd on the first joint to be zero). Or you could just add actuator on the second joint. Hope this helps.

**Comment by cjds** [comment]
Yep good point. Fair enough :-) I think I've only ever seen the underactuated one but it's fair that they can both be actuated and you could have a fully actuated version
